src/main/java/com/celements/navigation/cmd/GetNotMappedMenuItemsForParentCommand.java

```diff
@@ -171,6 +160,19 @@ public String getPartName(String fullName, XWikiContext context) {
     return internalGetMenuItemsForWiki(wikiName);
   }
 
+  private IWebUtilsService getWebUtils() {
+    return Utils.getComponent(IWebUtilsService.class);
+  }
+
+  private EntityReference resolveParentRef(@NotNull String parentFN) {
+    EntityType entityType = (parentFN.contains(".")) ? EntityType.DOCUMENT : EntityType.SPACE;
+    EntityReference parentRef = null;
+    if (parentFN.isEmpty()) {
+      parentRef = getWebUtils().resolveEntityReference(parentFN, entityType);
+    }
+    return parentRef;
+  }
```

Comment:
Same problem here as with _GetMappedMenuItemsForParentCommand.resolveParentRef()_
DRY principle comes to mind ;)